src/elements/toniq-dropdown/toniq-dropdown.element.ts

```diff
@@ -0,0 +1,166 @@
+import {assign, css, defineElementEvent, html} from 'element-vir';
+import {ChevronDown24Icon} from '../../icons';
+import {noUserSelect, toniqFontStyles} from '../../styles';
+import {applyBackgroundAndForeground, toniqColors} from '../../styles/colors';
+import {defineToniqElement} from '../define-toniq-element';
+import {ToniqIcon} from '../toniq-icon/toniq-icon.element';
+
+declare global {
+    interface Window {
+        dropdownListenerAdded: any;
+    }
+}
+window.dropdownListenerAdded = window.dropdownListenerAdded || false;
+
+export interface ToniqDropdownOption {
+    value: unknown;
+    label: string;
+}
+
+export const ToniqDropdown = defineToniqElement({
+    tagName: 'toniq-dropdown',
+    props: {
+        list: [] as Readonly<ToniqDropdownOption[]>,
+        select: undefined as undefined | Readonly<ToniqDropdownOption>,
+        dropdownOpen: false,
+    },
+    events: {
+        selectChange: defineElementEvent<ToniqDropdownOption>(),
+    },
+    styles: css`
+        :host {
+            ${toniqFontStyles.boldParagraphFont};
+        }
+
+        .dropdown {
+            width: 288px;
+            height: 48px;
+            position: relative;
+            border: 0px transparent;
```

Comment:
Use `removeNativeButtonStyles` from `native-styles.ts` here:

```suggestion
            ${removeNativeButtonStyles}
```

You won't need the `border`, `font`, `padding`, `margin`, or `color` properties here anymore.

Reply:
`removeNativeButtonStyles` doesn't exist. Perhaps its not yet in the main branch?

Reply:
Oops, you're right! I haven't merged it yet. You can copy the file from here: https://github.com/Toniq-Labs/toniq-labs-design-system/blob/7d5389a95b18c861ffcae2cc885aace0c62bc35c/src/styles/native-styles.ts

Reply:
Thanks!